commit 493330cb8cd73be8a598308b78f0fc1d4912843a
Author: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date:   Mon Apr 22 12:12:50 2013 -0400

    Fix kFreeBSD build
    
    Use general error numbers available on kFreeBSD.
    
    Reported-by: Petr Salinger <Petr.Salinger@seznam.cz>
    Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>

diff --git a/formats/ctf/ctf.c b/formats/ctf/ctf.c
index 4e4aabf..25f2caf 100644
--- a/formats/ctf/ctf.c
+++ b/formats/ctf/ctf.c
@@ -1037,7 +1037,7 @@ int ctf_open_trace_metadata_stream_read(struct ctf_trace *td, FILE **fp,
 	buflen = strlen(*buf);
 	if (!buflen) {
 		*fp = NULL;
-		return -ENODATA;
+		return -ENOENT;
 	}
 	*fp = babeltrace_fmemopen(*buf, buflen, "rb");
 	if (!*fp) {
